src/extension.ts

```diff
@@ -160,16 +152,19 @@ export async function deactivate(): Promise<void> {
 
 export async function updateTerraformStatusBar(documentUri: vscode.Uri): Promise<void> {
   const client = clientHandler.getClient();
-  const initSupported = clientHandler.clientSupportsCommand(`${client.commandPrefix}.terraform-ls.terraform.init`);
-  if (initSupported) {
-    const client = clientHandler.getClient();
-    const moduleUri = Utils.dirname(documentUri);
 
-    if (client) {
+  if (client) {
+    const initSupported = clientHandler.clientSupportsCommand(`${client.commandPrefix}.terraform-ls.terraform.init`);
+
+    if (initSupported) {
```

Comment:
Maybe use this chance to return early by inverting the if statement and reduce nesting?

Reply:
Yes! Good point